proto/checked/v1/BUILD

```diff
@@ -13,7 +13,6 @@ proto_library(
         "checked.proto",
     ],
     # cc_api_version = 2,
-    # go_api_version = 2,
```

Comment:
Hmm. We probably don't want to remove the go support.

Reply:
I did not remove this. This does not add in the original version.

Reply:
The go_proto_library target from the original source does not work with open source tooling ... further motivation for a split to using separate Bazel BUILD files for this project.